src/platform/nodejs/utils/httpUtils.ts

```diff
@@ -0,0 +1,79 @@
+import * as http from 'http';
+import * as https from 'https';
+import * as url from 'url';
+import { constants } from 'crypto';
+import { NodejsPlatformConfig } from '../../types';
+
+const tlsRestrictedAgent = new https.Agent(<any>{
```

Comment:
Can you add more details about this agent and the configured secure options?

Reply:
I think it is for users requiring special proxies or http agents to get around their configured firewalls, etc.